index.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style/style.css" type="text/css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <title>YouTube</title>
+    </head>
+    <body>
+        <!--Start of navigation bar-->
+        <header>
+            <nav>
+                <a href="#"><img class="bars" class="img" src="img/img_bar.png" alt="image bar"></a>
+                <a href="#"><img class="logo" src="img/youtube_logo.png" alt="YouTube"></a>
+                <input class="search" type="text" placeholder="Search" class="img">
+                <a href="#"><button id="loop"><i class="fa fa-search" aria-hidden="true"></i></button></a>
+                <a href="#"><img class="upload" class="img" src="img/img_arrow.png" alt="image arrow"></a>
+                <a href="#"><img class="black_square" class="img" src="img/black_square.png" alt="square image"></a>
+                <a href="#"><img class="bell" class="img" src="img/bell.png" alt="bell image"></a>
+                <a href="#"><img class="profile" src="img/images.jpg" alt=""></a>
+            </nav>
```

Comment:
The navigation is not being rendered as it should. Apart from the oversized element, all of the other elements are missing and the input search field should be centered. Please compare your clone with the original page to fix this.

| Source Page	| Clone Page |
|:---:|:---:|
|![Screen Shot 2020-07-14 at 1 06 22 AM](https://user-images.githubusercontent.com/48804258/87362185-3dac5500-c56e-11ea-9a23-872ef86252dd.png) |![Screen Shot 2020-07-14 at 1 06 56 AM](https://user-images.githubusercontent.com/48804258/87362212-51f05200-c56e-11ea-9fae-0afc02899c60.png)|